app/static/js/events/wizard/vue-app.js

```diff
@@ -166,13 +170,13 @@ var app = new Vue({
             }
         },
         addTicket: function (ticketType) {
-            var ticket = _.cloneDeep(TICKET);
```

Comment:
What's the reason behind adding ticket as a parameter in the vue-app instance itself ? javascript is pass by reference. Every time the ticket gets modified, `this.ticket` will also get modified. That's the reason there was a clone in the first place.